bindings/gumjs/gumv8symbol.cpp

```diff
@@ -261,6 +263,24 @@ GUMJS_DEFINE_FUNCTION (gumjs_symbol_find_functions_matching)
   g_free (str);
 }
 
+GUMJS_DEFINE_FUNCTION (gumjs_symbol_load)
+{
+  gchar * path;
+  if (!_gum_v8_args_parse (args, "s", &path))
+    return;
+
+  bool result;
```

Comment:
```suggestion
  bool success;
```

For consistency with the Duktape implementation. (And to clarify what the return value signifies.)